### DISCUSSION TITLE
Add "double click to reset to default value" for slider in menus

### DISCUSSION THREAD

**Discussion by Lombeelo** [ppy/osu#21505] [Ideas] [answered]
**Here's the main Idea:**
![image](https://user-images.githubusercontent.com/73432723/205462068-a1fba2ac-0dff-40c1-a74c-7dbded59320b.png)
![image](https://user-images.githubusercontent.com/73432723/205462092-f1d12ce4-0984-439b-a3fe-d128ac589dbe.png)
![image](https://user-images.githubusercontent.com/73432723/205462114-6d1f05f1-439f-4036-93af-17ca7eaad10a.png)
![image](https://user-images.githubusercontent.com/73432723/205462143-08f66d79-a753-4b4a-be0f-6a817bda1bdb.png)
Edit: This is very useful when you mess up with some settings and you don't know its defaults when it's time to undo what you did

**Comment by mk56-spn** [comment]
Just click the green light  to reset?

**Reply by ghost** [reply]
For me, it's a bit inconvenient to move my mouse over to the green light, especially since it's small. I'd very much prefer double-clicking the slider.

**Comment by peppy** [comment, marked as answer]
Can agree with adding this. It's a UX I use quite a lot in other applications.

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/pull/25163
